query/src/sql/plan_parser.rs

```diff
@@ -223,6 +222,23 @@ impl PlanParser {
         }))
     }
 
+    /// DfShowDatabase to plan
+    #[tracing::instrument(level = "info", skip(self, show), fields(ctx.id = self.ctx.get_id().as_str()))]
+    pub fn sql_show_databases_to_plan(&self, show: &DfShowDatabases) -> Result<PlanNode> {
+        let where_clause = match &show.where_opt {
+            Some(expr) => format!("WHERE {}", expr),
+            None => String::from(""),
+        };
+
+        self.build_from_sql(
+            format!(
+                "SELECT name FROM system.databases {} ORDER BY name",
```

Comment:
maybe "SELECT name AS Database FROM system.databases {} ORDER BY name" is better?

Reply:
Yeah, it really can make logic simple.

Reply:
> maybe "SELECT name AS Database FROM system.databases {} ORDER BY name" is better?  

It seems alias has not supported  in where clause for  datafuse now:)


```
datafuse :) SELECT name as Database FROM system.databases WHERE Database = 'ss' ORDER BY name;

SELECT name AS Database
FROM system.databases
WHERE Database = 'ss'
ORDER BY name ASC

Query id: e97d62da-80d8-4294-8da0-a086c0b1cc41


0 rows in set. Elapsed: 0.012 sec.

Received exception from server (version 2021.5.0):
Code: 6. DB::Exception: Received from 0.0.0.0:9001. DB:Exception. Unable to get field named "Database". Valid fields: ["name"].
```